### ISSUE TITLE
Unable to Connect to Power BI File

### ISSUE THREAD

**Issue by crooney2016** [DaxStudio/DaxStudio#67]
Updated - the issue was because I was using Power BI Desktop installed from the Windows Store. I have no issues when using the MSI installed version. 
----

Hello, I've installed the latest version of DAX Studio for the first time and am unable to connect to either an Excel or Power BI file even when the files are open. Even when I launch DAX studio from and Excel file with a Power Pivot model I am unable to load.

I get the message: "No running Power BI Desktop or SSDT Integrated Workspaces found."

![image](https://user-images.githubusercontent.com/32747635/38740368-fad5d1c4-3efc-11e8-8cf3-7a720c8533a3.png)

I get some Analysis Services fails on using the DAX Checker, but I am unsure why. This may be very simple, but I am new to this, so any help would be appreciated.  Thanks!

![image](https://user-images.githubusercontent.com/32747635/38740592-90d20f6c-3efd-11e8-9cd0-0505c1a8bfe0.png)

--- Copy of Full Message---
    Version =  v2.7.4.102

Checking Operating System
=======================
    OSCaption       = Microsoft Windows 10 Pro
    OSVersion       = 10.0.15063
    OSArchitecture  = 64-bit
    VisibleMemoryGB = 15.84
    FreeMemoryGB    = 8.90

    Culture Name              = en-US
    Culture DisplayName       = English (United States)
    Culture EnglishName       = English (United States)
    Culture 2-Letter ISO Name = en
    Culture DecimalSeparator  = .
    Culture GroupSeparator    = ,
    Culture CurrencySymbol    = $
    Culture ShortDatePattern  = M/d/yyyy

Checking .Net Framework
=======================
    v2.0.50727 -> 2.0.50727.4927
    v2.0.50727\1033 -> 2.0.50727.4927
    v3.0 -> 3.0.30729.4926
    v3.0\Setup -> 3.0.30729.4926
    v3.0\Setup\1033 -> 3.0.30729.4926
    v3.0\Setup\Windows Communication Foundation -> 3.0.4506.4926
    v3.0\Setup\Windows Presentation Foundation -> 3.0.6920.4902
    v3.5 -> 3.5.30729.4926
    v3.5\1033 -> 3.5.30729.4926
    v4\Client -> 4.7.02046
    v4\Client\1033 -> 4.7.02046
    v4\Full -> 4.7.02046
    v4\Full\1033 -> 4.7.02046
    v4.0\Client -> 4.0.0.0

Checking AMO
=======================
    FAIL > Could not load file or assembly 'Microsoft.AnalysisServices, Version=13.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
    PASS > Microsoft.AnalysisServices, Version=14.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91
    WARN > Could not load file or assembly 'Microsoft.AnalysisServices, Version=15.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

Checking ADOMD.NET
=======================
    FAIL > Could not load file or assembly 'Microsoft.AnalysisServices.AdomdClient, Version=13.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
    PASS > Microsoft.AnalysisServices.AdomdClient, Version=14.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91
    WARN > Could not load file or assembly 'Microsoft.AnalysisServices.AdomdClient, Version=15.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

Dax Studio Configuration
========================
    Path: "C:\Program Files\DAX Studio\DaxStudio.exe"
    Processing DaxStudio.exe.config file...
    Path: 'C:\Program Files\DAX Studio\DaxStudio.exe.config'
    Bindings Found: 9
    AMO : 13.0.0.0
    ADOMD : 13.0.0.0

Checking Excel Add-in
=======================
    Detected Excel Version: 16 - Excel_v16
    Excel Path: C:\PROGRA~2\MICROS~2\Office16\EXCEL.EXE
    Excel Architecture: x86

    DAX Studio Excel Add-in Registry keys
       Description: Dax Studio Excel Add-In
       FriendlyName: Dax Studio Excel Add-In
       Manifest: file:///C:/Program Files/DAX Studio/DaxStudio.vsto|vstolocal
       LoadBehavior: 3

    Checking for Excel 2010 Disabled Add-ins
      PASS > No Disabled items found.
    Checking for Excel 2013 Disabled Add-ins
      PASS > No Disabled items found.
    Checking for Excel 2016 Disabled Add-ins
      PASS > No Disabled items found.

    Attempting to load Excel Add-in: 
      PASS > Loaded Excel Add-in: DaxStudio, Version=2.7.4.102, Culture=neutral, PublicKeyToken=null

    Checking VSTO Configuration
    Architecture: x64
      v4
        Version 10.0.40820
        Install 1
        InstallerPath c:\Program Files\Common Files\Microsoft Shared\VSTO\10.0\VSTOInstaller.exe
      v4M
        VSTOMFeature_UNM 1
        VSTOMFeature_CLR35 1
        VSTOMFeature_CLR40 1
      v4R
        VSTORFeature_UNM 1
        VSTORFeature_CLR35 1
        VSTORFeature_CLR40 1
        Version 10.0.50903
        Revision 00
        ProductCode {9495AEB4-AB97-39DE-8C42-806EEF75ECA7}

**Comment by dgosbell** [comment]
This is likely to be 2 different issues. The method used to detect a PowerPivot model when running inside the Excel addin is completely different to the method used to find a running Power BI Desktop mode.

Were you using the same PBIX file both times? 

It should not make a difference how you installed Power BI Desktop, we updated our detection method shortly after the Store version released in v.2.7.1 so that it would work with both methods of installation. I just doubled check on a test VM that DAX Studio 2.7.4 could connect to the latest release of Store version of Power BI Desktop and it worked fine for me.
